tool/sdg/src/main/java/org/openscience/cdk/layout/RingPlacer.java

```diff
@@ -269,92 +292,44 @@ private void placeBridgedRing(IRing ring, IAtomContainer sharedAtoms, Point2d sh
+        logger.debug("placeBridgedRing->tStep: " + Math.toDegrees(tStep));
 
         double startAngle;
+        int direction = -1;
 
-        int direction = 1;
-        // if bond is vertical
-        if (xDiff == 0) {
-            logger.debug("placeBridgedRing->Bond is vertical");
-            //starts with the lower Atom
-            if (bondAtom1.getPoint2d().y > bondAtom2.getPoint2d().y) {
-                startAtom = bondAtom1;
-            } else {
-                startAtom = bondAtom2;
-            }
-
-            //changes the drawing direction
-            if (centerX < sharedAtomsCenter.x) {
-                direction = 1;
-            } else {
-                direction = -1;
-            }
-        }
-
-        // if bond is not vertical
-        else {
-            //starts with the left Atom
-            if (bondAtom1.getPoint2d().x > bondAtom2.getPoint2d().x) {
-                startAtom = bondAtom1;
-            } else {
-                startAtom = bondAtom2;
-            }
-
-            //changes the drawing direction
-            if (centerY - sharedAtomsCenter.y > (centerX - sharedAtomsCenter.x) * yDiff / xDiff) {
-                direction = 1;
-            } else {
-                direction = -1;
-            }
-        }
-        startAngle = GeometryUtil.getAngle(startAtom.getPoint2d().x - ringCenter.x, startAtom.getPoint2d().y
-                - ringCenter.y);
+        startAngle = GeometryUtil.getAngle(bondAtom1.getPoint2d().x - ringCenter.x, bondAtom1.getPoint2d().y - ringCenter.y);
 
-        IAtom currentAtom = startAtom;
+        IAtom currentAtom = bondAtom1;
         IBond currentBond = sharedAtoms.getConnectedBondsList(currentAtom).get(0);
 
-        Vector atomsToDraw = new Vector();
+        List<IAtom> atoms = new ArrayList<>();
         for (int i = 0; i < ring.getBondCount(); i++) {
             currentBond = ring.getNextBond(currentBond, currentAtom);
             currentAtom = currentBond.getConnectedAtom(currentAtom);
             if (!sharedAtoms.contains(currentAtom)) {
-                atomsToDraw.addElement(currentAtom);
+                atoms.add(currentAtom);
             }
         }
-        try {
-            logger.debug("placeBridgedRing->atomsToPlace: " + AtomPlacer.listNumbers(molecule, atomsToDraw));
-            logger.debug("placeBridgedRing->startAtom is: " + (molecule.getAtomNumber(startAtom) + 1));
-            logger.debug("placeBridgedRing->startAngle: " + Math.toDegrees(startAngle));
-            logger.debug("placeBridgedRing->addAngle: " + Math.toDegrees(addAngle));
-        } catch (Exception exc) {
-            logger.debug("Caught an exception while logging in RingPlacer");
-        }
 
-        addAngle = addAngle * direction;
-        atomPlacer.populatePolygonCorners(atomsToDraw, ringCenter, startAngle, addAngle, radius);
+        logger.debug("placeBridgedRing->atomsToPlace: " + AtomPlacer.listNumbers(molecule, atoms));
```

Comment:
You can use a comma instead of a "+" here, which makes the code run faster when debugging is turned of:

logger.debug("placeBridgedRing->atomsToPlace: ", AtomPlacer.listNumbers(molecule, atoms));

It saves a bit of time but mostly a unneeded extra String in that case.

(This can be cleaned up later... it won't block merging the patch in...)